group_vars/rhcs.yml.sample

```diff
@@ -712,7 +712,7 @@ prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.1
 #prometheus_user_id: '65534'  # This is the UID used by the prom/prometheus container image
 #prometheus_port: 9092
 #prometheus_conf_overrides: {}
-alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:4.1
+alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:4.6
```

Comment:
@guits here also registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6 for alertmanager with version 0.21.0 you have to change these images